tests/selenium/pages/treeherder.py

```diff
@@ -56,13 +56,13 @@ def clear_filter(self, method='pointer'):
         if method == 'keyboard':
             self._keyboard_shortcut(Keys.CONTROL + Keys.SHIFT + 'f')
         else:
-            self.selenium.find_element(*self._clear_filter_locator).click()
+            self.driver.find_element(*self._clear_filter_locator).click()
```

Comment:
This can be changed to `self.find_element` as PyPOM already offers an abstraction here.

Reply:
Ah even better, thank you. Updated :-)